terraspace/app/stacks/publishing/tfvars/us-west-2/prod.tfvars

```diff
@@ -0,0 +1,15 @@
+provider_ads_bucket_name = "ipfs-advertisement"
```

Comment:
run `terraform fmt`

Reply:
It doesn't work for tfvars file.. But fixed the formatting manually!